Automattic/jetpack
# Admin: use minimzed CSS for stats widget

Fixes #22670

#### Changes proposed in this Pull Request:

* Use minimzed version for dashboard-widget.css. It was probably automatically created started with #23840 but we hadn't updated the enqueue to use it.

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Connect Jetpack, inspect wp-admin to see if dashboard-widget.css (dev site) or dashboard-widget.min.css (production) is loaded.
